### ISSUE TITLE
Callback is called 3 times on start service [IOS]

### ISSUE THREAD

**Issue by dacianf** [rekab-app/background_locator#84]
@mehdok I've observe on ios that the callback function is called 3 times every time I start the service, after that everything works fine. I've commented line 101 from "BackgroundLocatorPlugin.m" which caches some locations. Afterwards, I observed that when the service is initialised it calls the callback for each cached location. Is that caching really necessary?

**Comment by mehdok** [comment]
No, it's not necessary and iOS part of this plugin needs a complete review. Sadly I don't have enough time to do that right now, but it's in the plan;
